src/python/pants/notes/master.rst

```diff
@@ -4,6 +4,199 @@ Master Pre-Releases
 This document describes development releases which occur weekly from master, and which have
 not undergone the vetting associated with ``stable`` releases.
 
+1.27.0.dev0 (3/14/2020)
+-----------------------
+
+API Changes
+~~~~~~~~~~~
+
+* Add `PythonBinary`, `PythonLibrary`, and `PythonTests` to the Target API (#9301)
+  `PR #9301 <https://github.com/pantsbuild/pants/pull/9301>`_
+
+* Store `Address` on V2 `Target` and pass it to `Field`s during validation (#9300)
+  `PR #9300 <https://github.com/pantsbuild/pants/pull/9300>`_
+
+* Remove deprecated `pants.backend.python` files moved to `pants.python` (#9238)
+  `PR #9238 <https://github.com/pantsbuild/pants/pull/9238>`_
+
+* Remove deprecated `--target-spec-file` and `--positional-arg-file` in favor of `--spec-file` (#9237)
+  `PR #9237 <https://github.com/pantsbuild/pants/pull/9237>`_
+
+* Remove deprecated `--owner-of` in favor of file arguments (#9235)
+  `PR #9235 <https://github.com/pantsbuild/pants/pull/9235>`_
+
+* Remove deprecated `--dependencies-internal-only` and `--dependencies-external-only` (#9258)
+  `PR #9258 <https://github.com/pantsbuild/pants/pull/9258>`_
+
+* Upgrade to pex 2.1.6. (#9255)
```

Comment:
Consider moving this higher in the section. The removals are all planned, whereas this is a new change. Being higher up makes it easier for folks to notice the change.